pkg/inspector/image-inspector.go

```diff
@@ -649,3 +622,81 @@ func createOutputDir(dirName string, tempName string) (string, error) {
 	}
 	return dirName, nil
 }
+
+// acquireImage returns error and iiapi.FilesFilter for this image.
+func (i *defaultImageInspector) acquireImage(client DockerRuntimeClient) (error, iiapi.FilesFilter) {
+	var err error
+	if len(i.opts.Container) == 0 {
```

Comment:
@enoodle this method calls for a refactoring, anything that looks like

```Golang
func ... {
    if (...) {
        ...
    } else {
        ...
    }
}
```

should be better designed.

Anyway it's not for this PR.

Reply:
@enoodle to be clear you can just factor out those large bodies to other functions, etc. what I am saying is that a whole long function with two in-line code paths should be refactored.